test/role-block/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
This needs to be integrated into tests. Meaning there should be a test that runs this and then checks that there's no errors of a specific type.

Reply:
I'm a bit lost as to what needs to be done, I've read `contributing.md` but that doesn't really tell me much as to how I'm supposed to create my test and push my code. Any documentation I migh have missed that would help me? 
Thanks

Reply:
Sorry, I missed your comment. @ssbarnea has created a reproducer test @ #793. Also, I've added a few references on adding pytest based tests @ https://github.com/ansible/ansible-lint/issues/725.

Reply:
@webknjaz You can close this one now.